packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -49,27 +47,69 @@ const metadata: Meta<MenuButtonArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description: 'The text content of the button.',
+            table: { category: 'slots' }
+        },
+        icon: {
+            name: 'start',
+            description:
+                'Set `slot="start"` to include an icon before the text content. This icon will be hidden when `content-hidden` is set.',
+            table: { category: 'slots' }
+        },
+        endIcon: {
+            name: 'end',
+            description: 'Set \`slot="end"\` to include an icon after the text content. This icon will be hidden when \`content-hidden\` is set.',
+            table: { category: 'slots' }
+        },
+        menu: {
+            description: 'The `nimble-menu` to be displayed when the button is toggled.',
+            table: { category: 'slots' }
+        },
         appearance: {
             options: Object.keys(ButtonAppearance),
             control: { type: 'radio' },
-            description: appearanceDescription
+            description: appearanceDescription,
+            table: { category: 'attributes' }
         },
         appearanceVariant: {
             name: 'appearance-variant',
             options: Object.keys(ButtonAppearanceVariant),
             control: { type: 'radio' },
-            description: appearanceVariantDescription
+            description: appearanceVariantDescription,
+            table: { category: 'attributes' }
         },
-        icon: {
-            description:
-                'When including an icon, set `slot="start"` on the icon to ensure proper styling.'
+        open: {
+            control: { type: 'boolean' },
+            description: 'Opens the menu.',
+            table: { category: 'attributes' }
         },
-        endIcon: {
-            description: endIconDescription
+        disabled: {
+            control: { type: 'boolean' },
+            description: 'Disables the button.',
+            table: { category: 'attributes' }
+        },
+        contentHidden: {
+            name: 'content-hidden',
+            description: 'Hides the default text content. Set to `true` when an icon is used as the primary content',
+            table: { category: 'attributes' }
         },
         menuPosition: {
+            name: 'menu-position',
+            description: 'The position of the menu relative to the button.',
             options: Object.values(MenuButtonPosition),
-            control: { type: 'radio' }
+            control: { type: 'radio' },
+            table: { category: 'attributes' }
+        },
+        toggle: {
```

Comment:
When you hover over the control area for these it shows "setup control".
![image](https://github.com/ni/nimble/assets/1588923/1945176d-224d-4270-a8fe-f668748e55be)

Is that avoided when setting `control: false`? 
https://storybook.js.org/docs/essentials/controls#disable-controls-for-specific-properties